packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -103,34 +127,44 @@ Search.propTypes.renderers = React.PropTypes.shape({
 });
 
 function Help({ renderers, ...props }) {
-	const Components = getRenderers(renderers);
+	const global = {
+		bsStyle: 'link',
+		icon: 'talend-question-circle',
+		label: 'Help',
+		tooltipPlacement: 'bottom',
+		...props,
```

Comment:
Previously, `...props` was placed after other props, so I think that `props` should be able to override fixed props.